src/providers/ogr/qgsogrprovider.h

```diff
@@ -189,15 +267,23 @@ class QgsOgrProvider : public QgsVectorDataProvider
     OGRLayerH ogrLayer;
     OGRLayerH ogrOrigLayer;
 
+    //! ogr get type
+    int mOgrGetType;
```

Comment:
This one should use an enumerated type. And would deserve comments. I'm not a super fan of the naming of the variable itself. It should probably reflect that fact that it is about the priority of the name on the id (if I've well understood the purpose) ?